--- netdata-1.3.0/plugins.d/node.d.plugin~	2016-09-04 23:05:48.000000000 +0300
+++ netdata-1.3.0/plugins.d/node.d.plugin	2016-09-04 23:05:55.090394196 +0300
@@ -1,13 +1,5 @@
-#!/usr/bin/env bash
-':' //; exec "$(command -v nodejs || command -v node || command -v js || echo "ERROR node.js IS NOT AVAILABLE IN THIS SYSTEM")" "$0" "$@"
-
-// shebang hack from:
-// http://unix.stackexchange.com/questions/65235/universal-node-js-shebang
-
-// Initially this is run as a shell script (#!/bin/sh).
-// Then, the second line, finds nodejs or node or js in the system path
-// and executes it with the shell parameters.
-
+#!/usr/bin/node
+// vim:ft=javascript
 // --------------------------------------------------------------------------------------------------------------------
 
 'use strict';
