--- netdata-1.0.0/plugins.d/node.d.plugin~	2016-03-22 22:36:06.000000000 +0200
+++ netdata-1.0.0/plugins.d/node.d.plugin	2016-04-06 14:04:17.294990309 +0300
@@ -1,13 +1,4 @@
-#!/bin/sh
-':' //; exec "$(command -v nodejs || command -v node || command -v js || echo "ERROR node.js IS NOT AVAILABLE IN THIS SYSTEM")" "$0" "$@"
-
-// shebang hack from:
-// http://unix.stackexchange.com/questions/65235/universal-node-js-shebang
-
-// Initially this is run as a shell script (#!/bin/sh).
-// Then, the second line, finds nodejs or node or js in the system path
-// and executes it with the shell parameters.
-
+#!/usr/bin/node
 // --------------------------------------------------------------------------------------------------------------------
 
 'use strict';
